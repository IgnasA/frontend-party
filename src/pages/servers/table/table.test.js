import React from 'react';
import { shallow } from 'enzyme';
import { createMockStore } from 'redux-test-utils';
import Header from './tableHeader';
import TableRow from './tableRow';

it('tableheader should match snapshot', () => {
  const wrapper = shallow(
    <Header
      direction="desc"
      handleSort={() => {}}
      handleDirection={() => {}}
    />,
  );
  expect(wrapper).toMatchSnapshot();
});

it('tablerow should match snapshot', () => {
  const wrapper = shallow(
    <TableRow
      measurement="km"
      rowData={[{ name: 'LT', distance: 10 }]}
    />,
  );
  expect(wrapper).toMatchSnapshot();
});
