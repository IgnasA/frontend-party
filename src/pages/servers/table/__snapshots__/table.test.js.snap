// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tableheader should match snapshot 1`] = `ShallowWrapper {}`;

exports[`tablerow should match snapshot 1`] = `ShallowWrapper {}`;
